uberenv.py

```diff
@@ -105,7 +105,7 @@ def parse_args():
                       default=None,
                       help="destination directory")
 
-    # what compiler to use
+    # what compiler to use and other package configurations
```

Comment:
We could also change the option description which is also inaccurate.
Maybe something like "Standard spack spec, but package name is optional".

Reply:
the description is now "standard spack spec"